src/oteltest/viz/__init__.py

```diff
@@ -105,8 +145,21 @@ def view_trace(self, filename):
             min_start = 0
         if max_end is None:
             max_end = 0
+
+        # Process Metrics 
```

Comment:
We might want to put this into its own function (and put process traces into a function). Once that's done maybe we can add a test for it.